www/hodl01/HW4/css/styles-grid.css

```diff
@@ -0,0 +1,86 @@
+* {
+    box-sizing: border-box;
+  }
+  body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+  }
+  .clear {
+    clear: both;
+  }
+  h1 {
+    text-align: center;
+    padding: 50px 10px;
+    color: white;
+    background-color: rgb(78, 146, 179);
+  }
+  .colors > input {
+    padding: 20px;
+    border: none;
+    font-size: 18px;
+    margin: 0;
+    float: left;
+    width: 25%;
+    color: white;
+  }
+  .instructions {
+    max-width: 1280px;
+    padding: 20px;
+    margin: 0 auto;
+    margin-bottom: 50px;
+  }
+  .container {
+    margin-bottom: 50px;
+    background-color: grey;
+  }
+  .float-box {
+    font-size: 50px;
+    text-align: center;
+    float: left;
+    color: white;
+    font-weight: bold;
+  }
+  .container-01 .float-box {
+    width: 100%;
+    padding: 100px;
+  }
+  .container-02 .float-box {
+    width: 50%;
+    padding: 100px;
+  }
+  .container-03 .float-box {
+    width: 33.33%;
+    padding: 100px;
+  }
+  .container-04 .float-box {
+    width: 25%;
+    padding: 50px;
```

Comment:
tyhle paddingy v .float-box to vsechno kazi, protoze jsou na vsech stranach, maji byt jen horni a dolni.

Reply:
opraveno